src/Adapters/MockAdapter.ts

```diff
@@ -134,6 +137,76 @@ export default class MockAdapter
         this.mockModels = (mockModelData as any) as DtdlInterface[];
         this.initializeMockTwinProperties();
     }
+    getVisualsOnPage(reportUrl: string, pageName: string): Promise<IVisual[]> {
+        if (!reportUrl || !pageName) {
+            return Promise.resolve([]);
+        }
+        // eslint-disable-next-line no-debugger
+        debugger;
+        if (pageName === 'page1') {
```

Comment:
should this be a constant that we can use in the test and in the adapter so it's clear there is a dependency there?